<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultFiberItem</class>
 <widget class="QWidget" name="ResultFiberItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ResultFiberItem</string>
  </property>
  <widget class="QLabel" name="lblBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>186</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>25</y>
     <width>292</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosX">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>88</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosZ">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>88</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosY">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>88</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRAN">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>133</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RAN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblARC">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>133</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ARC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFiberName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>83</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../org.gl.comm/resources/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
